docs/configuration.asciidoc

```diff
@@ -217,6 +217,26 @@ It also expects that a period (`.`) is used to separate the integer and the frac
 | 1.0     | Double 
 |============
 
+[float]
+[[config-transaction-max-spans]]
+==== `TransactionMaxSpans` (performance)
+
+Limits the amount of spans that are recorded per transaction. This is helpful in cases where a transaction creates a very high amount of spans (e.g. thousands of SQL queries). Setting an upper limit will prevent overloading the agent and the APM server with too much work for such edge cases.
```

Comment:
Can you please update this section to this (including the line breaks)
```
Limits the amount of spans that are recorded per transaction.
This is helpful in cases where a transaction creates a very high amount of spans,
for example, thousands of SQL queries.
Setting an upper limit helps prevent overloading the Agent and APM server in these edge cases.
```

Reply:
fixed